app/models/collection_version.rb

```diff
@@ -40,6 +40,13 @@ def accessioned?
       DepositCollectionJob.perform_later(collection_version)
     end
 
+    after_transition on: :update_metadata, new: :first_draft do |collection_version|
```

Comment:
Is the `on: :update_metadata,` necessary here? Doesn't the `new: :first_draft` already cover that transition?